packages/destination-actions/src/destinations/close/index.ts

```diff
@@ -0,0 +1,58 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+
+import createUpdateContactAndLead from './createUpdateContactAndLead'
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Close',
+  slug: 'close',
+  mode: 'cloud',
+
+  authentication: {
+    scheme: 'basic',
+    fields: {
+      api_key: {
+        label: 'Api key',
+        description: 'Your Close api key.',
+        type: 'string',
+        required: true
+      },
+      contact_custom_field_id_for_user_id: {
```

Comment:
Kinda weird that this is part of "authentication" but fine if it's supposed to work this way.

Reply:
Yea, pipedrive is same https://github.com/closeio/action-destinations/blob/87b1673f4d93ed66d1bc3a6cf4e4958e5a9c6099/packages/destination-actions/src/destinations/pipedrive/index.ts#L35